app/javascript/react/components/NewShopForm.js

```diff
@@ -73,81 +73,103 @@ const NewShopForm = (props) => {
     <form onSubmit={postNewShop}>
       <h2>Submit a new Bubble Tea Shop</h2>
       <ErrorList errors={errors} />
-      <label>
-        Shop Name
-        <input 
-          type="text"
-          name="name"
-          value={shopRecord.name}
-          onChange={handleInputChange}
-        />
-      </label>
-
-      <label>
-        Address
-        <input 
-          type="text"
-          name="address"
-          value={shopRecord.address}
-          onChange={handleInputChange}
-        />
-      </label>
-
-      <label>
-        City
-        <input 
-          type="text"
-          name="city"
-          value={shopRecord.city}
-          onChange={handleInputChange}
-        />
-      </label>
-
-      <label>
-        State
-        <input 
-          type="text"
-          name="state"
-          value={shopRecord.state}
-          onChange={handleInputChange}
-        />
-      </label>
-
-      <label>
-        Zip Code
-        <input 
-          type="text"
-          name="zip"
-          value={shopRecord.zip}
-          onChange={handleInputChange}
-        />
-      </label>
-
-      <label>
-        Telephone
-        <input 
-          type="text"
-          name="telephone"
-          value={shopRecord.telephone}
-          onChange={handleInputChange}
-        />
-      </label>
-
-      <label>
-        Website
-        <input 
-          type="text"
-          name="website"
-          value={shopRecord.website}
-          onChange={handleInputChange}
-        />
-      </label>
-
-      <div className="button-group">
-        <input 
-          className="button"
-          type="submit"
-        />
+
+      <div className="grid-container form-container">
+        <div className="grid-x grid-padding-x">
+          <div className="small-12 medium-6 cell">
```

Comment:
`small-12` may not be required for all these sections - by default things will want to span across the whole page, which is the 12 columns. By specifying `medium-6` says "on a screen size of medium or larger, use 6 columns"